SphinxDocs/post_process.py

```diff
@@ -0,0 +1,8 @@
+import re
+import glob
+
+number_regex = re.compile(r'^\d[\.\d]+ ', flags=re.M)
+for f in glob.glob('source/Manual/*.rst'):
```

Comment:
> html files to rst format and how error prone this is?

Although pandoc is not foolproof, it's pretty good. With basic regex, I could easily modify and make it better.

Reply:
Note: After automatic conversion, I did some manual editing too myself.